bats/fb-content-katello.bats

```diff
@@ -491,3 +483,7 @@ setup() {
   hammer docker manifest list --content-view-version-id=$cvv_id --fields="schema version,digest,tags" \
     --order='tag' | grep 'sha256:a6ecbb1553353a08936f50c275b010388ed1bd6d9d84743c7e8e7468e2acd82e'
 }
+
+@test "cleanup subscription-manager after content tests" {
```

Comment:
Sadly we can't rely on it (bats in EPEL7 doesn't support it), but there is a `teardown_file` which would be nicer for this.

Reply:
TIL, one day… :)